apps/src/StudioApp.js

```diff
@@ -1212,7 +1214,7 @@ StudioApp.prototype.showInstructionsDialog_ = function (level, autoClose, showHi
 
   var hideFn = _.bind(function () {
     // Momentarily flash the instruction block white then back to regular.
-    if ($(endTargetSelector).length) {
+    if ($(endTargetSelector).length && !instructionsInTopPane) {
```

Comment:
In fact, I don't think we even need the `$(endTargetSelector).length` check, since the way jQuery works if you do `$('#non_existent_element').css()` it's just a no-op.

That said, I think these these sort of performance optimizations are going to result in trivial amounts of difference, and my guidance would generally be to optimize for code readability over eking out these micro-optimizations.